feathr_project/test/test_pyduf_preprocessing_e2e.py

```diff
@@ -2,20 +2,19 @@
 from datetime import datetime, timedelta
 from pathlib import Path
 from pyspark.sql.functions import col,sum,avg,max
-from feathr.job_utils import get_result_df
-from feathr.anchor import FeatureAnchor
-from feathr.dtype import STRING, BOOLEAN, FLOAT, INT32, ValueType
-from feathr.feature import Feature
-from feathr.query_feature_list import FeatureQuery
-from feathr.settings import ObservationSettings
-from feathr.source import INPUT_CONTEXT, HdfsSource
-from feathr.typed_key import TypedKey
-from feathr.transformation import WindowAggTransformation
+from feathr.utils.job_utils import get_result_df
+from feathr.feature_definition.anchor import FeatureAnchor
+from feathr.feature_definition.dtype import STRING, BOOLEAN, FLOAT, INT32, ValueType
+from feathr.feature_definition.feature import Feature
+from feathr.feature_definition.query_feature_list import FeatureQuery
+from feathr.feature_definition.settings import ObservationSettings
+from feathr.feature_definition.source import INPUT_CONTEXT, HdfsSource
+from feathr.feature_definition.typed_key import TypedKey
+from feathr.feature_definition.transformation import WindowAggTransformation
```

Comment:
the convention is not to use underscore. 
definition should be enough.

this changes the docs. I am not sure how we can show that users only need to import feathr.xyz in the readthedocs.